packages/askar/package.json

```diff
@@ -32,14 +32,16 @@
     "tsyringe": "^4.8.0"
   },
   "devDependencies": {
-    "@hyperledger/aries-askar-nodejs": "^0.1.0",
-    "@hyperledger/aries-askar-shared": "^0.1.0",
+    "@hyperledger/aries-askar-nodejs": "^0.2.0-dev.1",
+    "@hyperledger/aries-askar-shared": "^0.2.0-dev.1",
     "@types/bn.js": "^5.1.0",
+    "@types/ref-array-di": "^1.2.6",
```

Comment:
Why do we need to add these types? Should they be added as deps to the shared components packages so we don't have to re-add them here?

Reply:
Yes, it seems that we should add them as dependency of either shared components of the patched ffi-napi/ref-napi (most likely these ones).